examples/fluent-bit/amazon-opensearch-serverless/aoss-data-access-policy.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+       "Rules":[
+          {
+             "ResourceType":"index",
+             "Resource":[
+                "index/_______MY_COLLECTION_NAME_______/*"
```

Comment:
just checking, why the ______ ? that's just to call out that you need to replace it with your collection name?

when I input my collection name it shouldn' have the ______

Sorry I know this might be annoying but some user may actually think this.

Reply:
It was because I wanted to call out that "index/" is doesn't need to be changed, but "__MY_COLLECTION_NAME__" does. I was thinking someone might feel like they need to replace index with their specific index if the ____MY_COLLECTION_NAME__ didn't look different.

Regardless, I updated it to index/MY_COLLECTION_NAME/* to resolve the issue you pointed out